conjur/cli.py

```diff
@@ -28,14 +31,16 @@ class Cli():
     Main wrapper around CLI-like usages of this module. Provides various
     helpers around parsing of parameters and running client commands.
     """
+    LOGGING_FORMAT = '%(asctime)s %(levelname)s: %(message)s'
 
     @staticmethod
-    def main_description():
+    def usage(args):
         """
         This method builds the header for the main screen.
         """
         return '''Usage:
-  conjur [global options] command [subcommand] [value] [--option=arg]'''
+  {}'''.format(args)
```

Comment:
What this line does?

Reply:
args is basically the `conjur [global options] command [subcommand] [value] [--option=arg]` you see above. I expect to get different usages for each command so made it a function and just sub args in

Reply:
this usage was supplied to us by UX